File: app/src/main/java/ntutifm/game/google/net/ApiClass/CitySpeed.java
package ntutifm.game.google.net.ApiClass;

import com.google.gson.annotations.SerializedName;

public class CitySpeed {

    @SerializedName("Direction")
    private String direction ="Direction";
    @SerializedName("AvgSpeed")
    private Double avgSpeed=0.0;
    @SerializedName("Volume")
    private Integer volume=-99;


    public String getDirection() {
        return this.direction;
    }
    public Double getAvgSpeed() {
        return this.avgSpeed;
    }
    public Integer getVolume() {
        return this.volume;
    }
}

File: app/src/main/java/ntutifm/game/google/apiClass/CitySpeed.kt
package ntutifm.game.google.apiClass

import com.google.gson.annotations.SerializedName

data class CitySpeed (
    @SerializedName("Direction") val direction : String,
    @SerializedName("AvgSpeed") val avgSpeed :Double,
    @SerializedName("Volume") val volume :Int = 99
)